packages/spawner/test/swingsetTests/contractHost/test-contractHost.js

```diff
@@ -1,9 +1,7 @@
 /* global __dirname */
-// TODO Remove babel-standalone preinitialization
-// https://github.com/endojs/endo/issues/768
-import '@agoric/babel-standalone';
-import '@agoric/install-ses';
-import test from 'ava';
+// eslint-disable-next-line import/order
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+// eslint-disable-next-line import/order
```

Comment:
`yarn lint-fix` keeps rearranging my `import` statements, I think it wants all "external" imports to appear in alphabetical order, followed by all "internal" imports in order, and this idiom seems to work to keep the `prepare-test-env-ava` as the very first one (since it has side-effects and won't work if it appears too late). I wouldn't be surprised if there is a way to phrase it with just a single override, but I haven't found it yet, and I got tired of fighting with eslint so I stopped once I got something that worked :).

Reply:
I think you may be running a different (newer) version of ESLint than I am. My version (which I believe is the agoric-sdk version, but I should check) doesn't enforce that. Here's the output after removing the eslint-disable:

<img width="990" alt="Screen Shot 2021-07-25 at 12 04 19 PM" src="https://user-images.githubusercontent.com/2441069/126910406-0e021913-c0ae-4358-a251-59f616b9e5fb.png">